### ISSUE TITLE
Implement method to retrieve followers/following for user

### ISSUE THREAD

**Issue by PittCaleb** [dmytrostriletskyi/threads-net#10]

**Comment by AmirHJabari** [comment]
Yes please this is the most useful thing for us programmers right now.
I need to create some scripts to get more followers
Thanks for your great work ❤️

**Comment by dmytrostriletskyi** [comment]
https://github.com/dmytrostriletskyi/threads-net#get-user-followers
https://github.com/dmytrostriletskyi/threads-net#get-user-following
